packages/forms/src/lib/mask/README.md

```diff
@@ -0,0 +1,43 @@
+# @acpaas-ui/ngx-components/forms
+
+With the @acpaas-ui mask module you can mask input fields.
+
+## Usage
+
+```typescript
+import { MaskModule } from '@acpaas-ui/ngx-components/mask'`;
+```
+
+## Documentation
+
+Visit our [documentation site](https://acpaas-ui.digipolis.be/) for full how-to docs and guidelines
+
+### API
+
+| Name         | Default value | Description |
+| -----------  | ------ | -------------------------- |
+| `@Input() auiMask: string;` | - | Use the auiMask input to apply config. |
```

Comment:
mask formaat uitleggen (of naar linken als er ergens online documentatie van is)